pkg/applicationmanager/controllers/applicationbackup.go

```diff
@@ -979,23 +1012,34 @@ func (a *ApplicationBackupController) backupVolumes(backup *stork_api.Applicatio
 			drivers := a.getDriversForBackup(backup)
 			volumeInfos := make([]*stork_api.ApplicationBackupVolumeInfo, 0)
 			for driverName := range drivers {
-
 				driver, err := volume.Get(driverName)
 				if err != nil {
 					return err
 				}
-
+				// skip fetching status for skipped vols
+				if skipDriver == driverName {
+					logrus.Tracef("skipping driver %v for status check", driverName)
+					continue
+				}
 				status, err := driver.GetBackupStatus(backup)
 				if err != nil {
-					return fmt.Errorf("error getting backup status for driver %v: %v", driverName, err)
+					// This will have status of failed volinfo whose status could not be got
+					// We need them also to be added to the list
+					logrus.Errorf("error getting backup status for driver %v: %v", driverName, err)
+					volumeInfos = append(volumeInfos, status...)
```

Comment:
In the case of err, we will be getting status ( volInfo list) as nil from driver.GetBackupStatus.  Do we need to append them ?

Reply:
If you check the definition of the GetBackupStatus for portworx ( for example), it will return nil as volumeInfo that means, you are not adding anything in the case of err. So we will need this append, I think.
https://github.com/libopenstorage/stork/blob/master/drivers/volume/portworx/portworx.go#L3438
https://github.com/libopenstorage/stork/blob/master/drivers/volume/portworx/portworx.go#L3456